src/core/qgis.h

```diff
@@ -2083,11 +2083,23 @@ class CORE_EXPORT Qgis
     enum class JoinStyle SIP_MONKEYPATCH_SCOPEENUM_UNNEST( QgsGeometry, JoinStyle ) : int
       {
       Round SIP_MONKEYPATCH_COMPAT_NAME( JoinStyleRound ) = 1, //!< Use rounded joins
-      Miter SIP_MONKEYPATCH_COMPAT_NAME( JoinStyleMiter ), //!< Use mitered joins
-      Bevel SIP_MONKEYPATCH_COMPAT_NAME( JoinStyleBevel ), //!< Use beveled joins
+      Miter SIP_MONKEYPATCH_COMPAT_NAME( JoinStyleMiter ),     //!< Use mitered joins
+      Bevel SIP_MONKEYPATCH_COMPAT_NAME( JoinStyleBevel ),     //!< Use beveled joins
```

Comment:
revert unnecessary whitespace change

Reply:
I will revert these changes. FYI this is linked to the use of 2 file formatters (astyle and clang-format). They are not really the same and generate many small changes...